<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/new/prefix1/pic/icon.png</normaloff>:/new/prefix1/pic/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:focus{padding: -1;} </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>311</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>搜索</string>
     </attribute>
     <widget class="QFrame" name="frame_func">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>311</width>
        <height>531</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>311</width>
         <height>71</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>路线查询</string>
        </attribute>
        <widget class="QLineEdit" name="line_start">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/pic/search.png);border-radius: 3px; color: rgb(126,126,126); </string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_search">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { border-image: url(:/new/prefix1/pic/appbar.foot.png); } QPushButton:hover {border-image: url(:/new/prefix1/pic/appbar.social.aim.png); }

</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="line_final">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/pic/search.png);border-radius: 3px; color: rgb(126,126,126); </string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/pic/appbar.arrow.right.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>推荐路线</string>
        </attribute>
        <widget class="QLineEdit" name="line_start_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/pic/search.png);border-radius: 3px; color: rgb(126,126,126); </string>
         </property>
        </widget>
        <widget class="QPushButton" name="Button_search_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>0</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {border-image: url(:/new/prefix1/pic/appbar.magnify.browse.png); } QPushButton:hover { border-image: url(:/new/prefix1/pic/appbar.magnify.png); }

</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QTextEdit" name="text_path">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/pic/bg.png);border-radius: 3px; color: rgb(126,126,126); </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(136, 136, 102); border-radius: 3px; color: rgb(255, 255, 255); </string>
       </property>
       <property name="text">
        <string> 景点介绍</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="text_intro">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>311</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/pic/bg.png);border-radius: 3px; color: rgb(126,126,126); </string>
       </property>
      </widget>
      <widget class="QFrame" name="img_frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>311</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/pic/psu.jpg);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>160</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/prefix1/pic/appbar.close.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_23">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>520</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>编辑</string>
       </property>
      </widget>
      <zorder>tabWidget</zorder>
      <zorder>text_path</zorder>
      <zorder>text_intro</zorder>
      <zorder>img_frame</zorder>
      <zorder>pushButton_23</zorder>
      <zorder>label_5</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>编辑</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(165,122, 139); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(245, 169, 234); }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>291</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="line_name">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>景点名：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>景点信息：</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="text_info">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="pic_path">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>路径请使用纯英文</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>景点图片：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_x">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_y">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_19">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>添加新顶点</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_del">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除顶点</string>
        </property>
       </widget>
       <widget class="QPushButton" name="open_pic">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>100</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>打开图片</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_pathedit">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>220</y>
         <width>241</width>
         <height>211</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="line_start_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>起点：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_final_edit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>终点：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_arc">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>边名：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>长度：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_length">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pathButton_del">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>删除边</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_25">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>添加边</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="open_bgpic">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>440</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>打开图片</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_bg">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>440</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>背景图片：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="savebg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>470</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存背景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>470</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>还原默认背景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>470</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1700</width>
      <height>1350</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>tabWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_19</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addnode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>962</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delnode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>852</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_25</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addpath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>1069</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_search</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>974</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathButton_del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delpath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>891</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_17</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>hide_f()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>982</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>1069</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_pic</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>add_pic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>962</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>993</x>
     <y>-95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ret()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>1273</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savebg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>1259</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savebg</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_bg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>765</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_bgpic</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_bg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>952</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>1225</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_search_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>recommend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>1061</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>savepic()</signal>
  <slot>introduct()</slot>
  <slot>search()</slot>
  <slot>hide_f()</slot>
  <slot>recommend()</slot>
  <slot>addnode()</slot>
  <slot>delnode()</slot>
  <slot>addpath()</slot>
  <slot>delpath()</slot>
  <slot>add_pic()</slot>
  <slot>ret()</slot>
  <slot>save_bg()</slot>
  <slot>open_bg()</slot>
 </slots>
</ui>
